### PR TITLE
aliyun-cli 3.0.74

### PR DESCRIPTION
**Pull request by chenrui333**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Moisan** [comment]
```
==> make metas
git submodule update
fatal: not a git repository (or any of the parent directories): .git
make: *** [deps] Error 128
```

**Comment by carlocab** [comment]
This can probably be made to use Go modules now. Not sure if it'll fix the `make metas` failure though, but maybe it won't be needed anymore after switching.

**Comment by Bo98** [comment]
I believe it will be necessary to switch to a git clone here.

**Comment by carlocab** [comment]
> I believe it will be necessary to switch to a git clone here.

Yes, looks like. But I'm usually suspicious of formulae that need Git checkouts because that's often how they vendor dependencies.

**Comment by Bo98** [comment]
This one is using submodules to pull in https://github.com/aliyun/aliyun-openapi-meta

File: Formula/aliyun-cli.rb [hunk 1]
```diff
@@ -13,17 +14,12 @@ class AliyunCli < Formula
...
-    ENV["PATH"] = "#{ENV["PATH"]}:#{buildpath}/bin"
-    (buildpath/"src/github.com/aliyun/aliyun-cli").install buildpath.children
-    cd "src/github.com/aliyun/aliyun-cli" do
-      system "make", "metas"
-      system "go", "build", "-o", bin/"aliyun", "-ldflags",
-                            "-X 'github.com/aliyun/aliyun-cli/cli.Version=#{version}'", "main/main.go"
-    end
+    system "make", "metas"
+    system "go", "build", "-o", bin/"aliyun", "-ldflags",
+          "-X 'github.com/aliyun/aliyun-cli/cli.Version=#{version}'", "main/main.go"
```
Comment (carlocab) [inline Formula/aliyun-cli.rb:22]:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-X github.com/aliyun/aliyun-cli/cli.Version=#{version}"),
                          "-o", bin/"aliyun", "main/main.go"
```

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**